File: src/listas.py
"""."""
lista = ["manzana", "naranja", "sandia"]

print(lista)
print(lista[2])

# Agregar elementos
lista.append("fruitlla")
lista = lista + ["pera", "arándanos"]
print(lista)

# Eliminar elementos
lista.remove("pera")
del lista[0]

# Consumir elemento del final
elemento = lista.pop()

# Consumir elemento del principio
elemento = lista.pop(0)

print(lista)

File: de_php_a_python/src/listas.php
<?php

$lista = ["manzana", "naranja", "sandia"];

var_dump($lista);
var_dump($lista[2]);

// Agregar elementos
$lista[] = "frutilla";
$lista = array_merge($lista, ["pera", "arándanos"]);
var_dump($lista);

// Eliminar elemento
$lista = array_diff($lista, ["pera"]);
unset($lista[0]);

// Consumir un elemento del final
$elemento = array_pop($lista);

// Consumir un elemento del principio
$elemento = array_shift($lista);

var_dump($lista);
